src/main/flight/alt_hold.c

```diff
@@ -133,14 +134,14 @@ void altHoldUpdateTargetAltitude(void)
 void altHoldUpdate(void)
 {
     // check if the user has changed the target altitude using sticks
-    if (altHoldConfig()->alt_hold_adjust_rate) {
+    if (altHoldConfig()->alt_hold_climb_rate) {
         altHoldUpdateTargetAltitude();
     }
     altitudeControl(altHold.targetAltitudeCm, taskIntervalSeconds, altHold.targetVelocity);
 }
 
 void updateAltHold(timeUs_t currentTimeUs) {
-    UNUSED(currentTimeUs);
+    UNUSED(currentTimeUs); 
```

Comment:
```suggestion
    UNUSED(currentTimeUs);
```

Reply:
Refactoring is minimal.  The only pure refactoring is abbreviation of autopilot to ap, as discussed previously but not actually implemented.  A lot of what 'seems like' refactoring is changes that are needed get the code to work as originally intended.